etc/kayobe/ansible/wazuh-manager.yml

```diff
@@ -125,3 +116,38 @@
       service:
         name: wazuh-dashboard
         state: restarted
+
+- hosts: wazuh-agent
```

Comment:
Should this play be in wazuh-agents.yml?

Reply:
Yes seems sensible, I've moved them over